packs/data/backgrounds.db/child-of-the-twin-village.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "0FgYkkKv9u8zxWiO",
+    "img": "systems/pf2e/icons/default-icons/background.svg",
+    "name": "Child of the Twin Village",
```

Comment:
I could swear I got this one made at least, maybe I just forgot to extract that compendium.